src/igvfd/searches/configs/curated_set.py

```diff
@@ -7,36 +7,54 @@
 def curated_set():
     return {
         'facets': {
-            'lab.title': {
-                'title': 'Lab'
-            },
-            'award.component': {
-                'title': 'Award'
-            },
-            'taxa': {
-                'title': 'Taxa'
+            'file_set_type': {
+                'title': 'File Set Type'
             },
             'assemblies': {
                 'title': 'Assemblies'
             },
             'transcriptome_annotations': {
                 'title': 'Transcriptome Annotations'
             },
+            'taxa': {
+                'title': 'Taxa'
+            },
```

Comment:
Look like the facet groups use both `taxa` and `donors.taxa`, so you need to add `donors.taxa` here.

Reply:
after looking at production data, i think we should use `taxa` instead of `donors.taxa`.